source/renderer/app/utils/validations.js

```diff
@@ -53,11 +56,14 @@ export const isUnicaseString = (password: string) =>
   every(password.split(''), (char) => isCaselessString(char));
 
 /**
- * Enforces passwords without spaces and a minimum of 10 characters.
+ * Enforces passwords without spaces and a minimum of 10 characters and a maximum of 255 characters.
  */
 export const isValidSpendingPassword = (password: string): boolean => {
   // Should contain at least 10 characters
-  return password.length >= 10;
+  return (
+    password.length >= MIN_PASSWORD_LENGTH &&
+    password.length <= MAX_PASSWORD_LENGTH
```

Comment:
We could remove this check since the `PasswordInput` defaults to `maxLength=255` so it always will meet this condition